docs/Userguide_login.rst

```diff
@@ -189,3 +189,11 @@ require a lot of resources: editing via VSCode, building conda environments, tes
 The ``mila-cpu`` entry should be in your ``.ssh/config``. Changes are at your own risk.
 While it is possible to tweak it to allocate a GPU, doing so will prevent simultaneous
 connections to it (until Slurm is upgraded to version 22.05 or later).
+
+Using a non-Bash Unix shell
+---------------------------
+
+While IT and Infra will not provide personalized support for non-Bash Unix
```

Comment:
Would `1-on-1 support` be better? The idea is that users would not request help to setup their stuff with Zsh, but still know that it should work and for the team to know if things break

Reply:
Message updated